packages/react/src/primitives/Checkbox/Checkbox.tsx

```diff
@@ -21,11 +22,11 @@ const CheckboxPrimitive: Primitive<CheckboxProps, 'input'> = (
     defaultChecked,
     hasError,
     isDisabled,
+    isIndeterminate,
```

Comment:
Will this conflict with any existing props? For example `defaultChecked`

Reply:
It complies with native HTML spec. `indeterminate` state will always have the precedence over other states.